website/client-old/js/config.js

```diff
@@ -35,7 +35,7 @@ angular.module('habitrpg')
           if (isUserLoaded && notification.type === 'CRON') {
             // If the user is already loaded, do not show the notification, syncing will show it
             // (the user will be synced automatically)
-            $rootScope.User.readNotification(notification.id);
+            // $rootScope.User.readNotifications([notification.id]);
```

Comment:
No I think the point of this code was another. If you're online and your cron time passes, the next time you complete and action two things happens:

1. The notification about cron is sent to the browser
2. the browser notices that the user has been updated (user._v changed) and issue a sync request and if there's a difference between the local user data and what the server sent (for example because hp changed after cron) it'll show another notification for the HP change

So this code is there to avoid showing the same notification twice. Also if it's there it's for a reason and before removing existing code I would like us to check whether it's used or not. Also because commenting it leaving the comments above in place and an empty IF condition doesn't make sense and only increase confusion for the next person looking at the code